specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-11-01-preview/examples/RegistryList.json

```diff
@@ -0,0 +1,65 @@
+{
+  "parameters": {
+    "api-version": "2023-11-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.ContainerRegistry/registries/myRegistry",
+            "name": "myRegistry",
+            "type": "Microsoft.ContainerRegistry/registries",
+            "location": "westus",
+            "tags": {
+              "key": "value"
+            },
+            "sku": {
+              "name": "Standard",
+              "tier": "Standard"
+            },
+            "properties": {
+              "loginServer": "myRegistry.azurecr-test.io",
+              "creationDate": "2021-06-15T21:38:26.1537861Z",
+              "provisioningState": "Succeeded",
+              "adminUserEnabled": false,
+              "networkRuleSet": {
+                "defaultAction": "Allow",
+                "ipRules": []
+              },
+              "policies": {
+                "quarantinePolicy": {
+                  "status": "disabled"
+                },
+                "trustPolicy": {
+                  "type": "Notary",
+                  "status": "disabled"
+                },
+                "retentionPolicy": {
+                  "days": 7,
+                  "lastUpdatedTime": "2021-06-15T21:40:12.8506835+00:00",
+                  "status": "disabled"
+                },
+                "exportPolicy": {
+                  "status": "enabled"
+                }
+              },
+              "encryption": {
+                "status": "disabled"
+              },
+              "status": null,
+              "dataEndpointEnabled": false,
+              "dataEndpointHostNames": [],
+              "privateEndpointConnections": [],
+              "publicNetworkAccess": "Enabled",
+              "networkRuleBypassOptions": "AzureServices",
+              "zoneRedundancy": "Disabled",
+              "anonymousPullEnabled": false
```

Comment:
Please add the new property metadataSearchEnabled into the example response and request body.